packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/SuccessStep/SuccessStep.tsx

```diff
@@ -1,9 +1,26 @@
 import React from 'react';
 
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWButton } from 'views/components/component_kit/new_designs/cw_button';
 import './SuccessStep.scss';
 
 const SuccessStep = () => {
-  return <div className="SuccessStep">SuccessStep</div>;
+  return (
+    <div className="SuccessStep">
+      <img src="../../static/img/communityIslive.png" alt="" className="img" />
+      <CWText type="h2">Your community is live!</CWText>
+      <CWText type="b1">
+        Your community is now live and discoverable on Common. You can now use
```

Comment:
I think this is gray color on the designs. Yours is black

Reply:
done